template/workitem.update/body.html

```diff
@@ -48,11 +48,11 @@
             border: 1px solid #d7d7d7;">
       </div>
       <p>
-        <img src="{{sizeImage .workitemOwner.Data.Attributes.ImageURL 22}}" style="height: 22px; width: 22px; vertical-align: middle;"
+        <img src="{{sizeImage .actor.Data.Attributes.ImageURL 22}}" style="height: 22px; width: 22px; vertical-align: middle;"
```

Comment:
@jarifibrahim changes looks good.  Have you run `make preview` and see email_output_html before/after your changes.  `make preivew` run `./preview/main.go` file which use email_template and generate email_output_html file at `./tmp/`.

Reply:
>Have you run make preview and see email_output_html before/after your changes. make preivew run ./preview/main.go file which use email_template and generate email_output_html file at ./tmp/.

It won't work because `I cannot test this change locally because the actor value is dependent on the token in the request but make preview doesn't use the actual tokens.`